sdv/multi_table/base.py

```diff
@@ -200,13 +200,10 @@ def get_metadata(self, version='original'):
             error_msg = f"Unrecognized version '{version}', please use 'original' or 'modified'."
             raise ValueError(error_msg)
 
-        if version == 'original':
-            original = (
-                self._original_metadata if hasattr(self, '_original_metadata') else self.metadata
-            )
-            return original
-        else:
-            return self.metadata
+        if version == 'original' and hasattr(self, '_original_metadata'):
+            return deepcopy(self._original_metadata)
```

Comment:
Is it possible this is a `SingleTableMetadata` for the backwards compatibility case? If so, should it be converted to `Metadata`? Same for line 206.

Reply:
I guess thats why we had it with `Metadata.load_from_dict` ? 😓 I will just re-vert this changes (not using deepcopy).